recipes/orange3/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: orange3
+  version: 3.3.6
+
+source:
+  git_url: https://github.com/biolab/orange3.git
+  git_tag: 3.3.6
+
+build:
+  number: 0
+  entry_points:
+    - orange-canvas = Orange.canvas.__main__:main
+  osx_is_app: True
```

Comment:
Current error is `Orange requires Python >= 3.4` so need to add a `skip` selector here.